test/REQUIRE

```diff
@@ -1 +1,14 @@
 Clp
```

Comment:
Maybe remove `REQUIRE` for tests, as dependencies are now handled in `Project.toml`

Reply:
removed